contributing.md

```diff
@@ -43,19 +43,28 @@ Before starting major work, you should touch base with the maintainers of Featur
 * You will need to install Spark, Scala, and GraphViz to run all unit tests:
 
      **macOS** (use [Homebrew](https://brew.sh/)):
-     ```bash
+     ```console
      brew tap AdoptOpenJDK/openjdk
      brew install --cask adoptopenjdk11
-     brew install scala
-     brew install apache-spark
```

Comment:
why are these removed? I think they are need for tests to complete

Reply:
We changed it to one command
`brew install scala apache-spark`

Reply:
Oh right. @jeff-hernandez We should change the above command to `brew install scala apache-spark graphviz`

Reply:
updated